docs/data/joy/getting-started/tutorial/LoginFinal.js

```diff
@@ -35,59 +35,56 @@ function ModeToggle() {
 
 export default function LoginFinal() {
   return (
-    <CssVarsProvider>
```

Comment:
This is duplicated, global styles are injected twice in the page, making a mess in the dev tools to debug.